x-pack/plugins/observability_solution/synthetics/public/apps/synthetics/components/common/alerting_callout/alerting_callout.tsx

```diff
@@ -51,111 +65,276 @@ export const AlertingCallout = ({ isAlertingEnabled }: { isAlertingEnabled?: boo
+      <EuiSpacer size="m" />
+    </>
+  );
+};
 
-  if ((!missingConfig && !missingRules) || isHidden) {
+const MissingRuleCallout = ({
+  showCallout,
+  setFlyoutVisibleArgs,
+  title,
+  buttonLabel,
+  content,
+  manageRulesProps,
+  createRuleTestId,
+  enableDefaultRuleTestId,
+  defaultRuleType,
+}: MissingRuleCalloutProps) => {
+  const dispatch = useDispatch();
+  const showFlyout = useCallback(
+    () => dispatch(setAlertFlyoutVisible(setFlyoutVisibleArgs)),
+    [dispatch, setFlyoutVisibleArgs]
+  );
+  const dispatchEnableDefaultRule = useCallback(() => {
+    dispatch(
+      enableDefaultAlertingAction.get({
+        enableTls: defaultRuleType === SYNTHETICS_TLS_RULE,
+        enableMonitorStatus: defaultRuleType === SYNTHETICS_STATUS_RULE,
+      })
+    );
+  }, [dispatch, defaultRuleType]);
+  if (!showCallout) {
     return null;
   }
-  const point = missingRules === missingConfig ? '* ' : '';
-
-  const configCallout = missingConfig ? (
-    <EuiMarkdownFormat>{`${point}${MISSING_CONFIG_LABEL}`}</EuiMarkdownFormat>
-  ) : null;
-
-  const rulesCallout = missingRules ? (
-    <EuiMarkdownFormat>{`${point}${MISSING_RULES_PRIVILEGES_LABEL}`}</EuiMarkdownFormat>
-  ) : null;
-
   return (
-    <EuiCallOut
-      title={
-        <FormattedMessage
-          id="xpack.synthetics.alerting.noConnectorsCallout.header"
-          defaultMessage="Alerts are not being sent"
-        />
-      }
-      color="warning"
-      iconType="warning"
-    >
-      {configCallout}
-      {rulesCallout}
-      {missingConfig && (
-        <>
-          <EuiSpacer size="m" />
-          <EuiButton
-            data-test-subj="syntheticsAlertingCalloutLinkButtonButton"
-            href={url}
-            color="warning"
-          >
-            <FormattedMessage
-              id="xpack.synthetics.alerting.noConnectorsCallout.button"
-              defaultMessage="Configure now"
-            />
-          </EuiButton>
-        </>
-      )}
-      <EuiButtonEmpty
-        data-test-subj="syntheticsMissingRulesCalloutRemindMeLaterButton"
-        onClick={() => {
-          setIsHidden(true);
-        }}
-      >
-        <FormattedMessage
-          id="xpack.synthetics.alerting.remindMeLater.button"
-          defaultMessage="Remind me later"
-        />
-      </EuiButtonEmpty>
-    </EuiCallOut>
+    <>
+      <EuiCallOut title={title} color="warning" iconType="warning">
+        {content}
+        <EuiSpacer size="xs" />
+        <EuiFlexGroup>
+          <EuiFlexItem />
+          <EuiFlexItem grow={false}>
+            <EuiButton data-test-subj={createRuleTestId} onClick={showFlyout} color="warning">
+              {buttonLabel}
+            </EuiButton>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="none" alignItems="baseline">
+              <EuiFlexItem>
+                <EuiIconTip
+                  color="warning"
+                  content="Enable default rule"
```

Comment:
i18n and also we'll need more description. i'm sure this is a todo for you, just don't want you to forget. :)

Reply:
b9632f5b3b3b219200cfa62a43ebeddb77ff8335